indico/modules/events/templates/display/conference/base.html

```diff
@@ -5,19 +5,23 @@
 
 {%- macro _format_event_date(event) -%}
     {% set start_dt = event.start_dt_display.astimezone(event.display_tzinfo) %}
+    {% set start_dt_iso = event.start_dt_display.isoformat() %}
     {% set end_dt = event.end_dt_display.astimezone(event.display_tzinfo) %}
+    {% set end_dt_iso = event.end_dt_display.isoformat() %}
     {% if start_dt.date() == end_dt.date() %}
-        {{ start_dt | format_date('long') }}
+       <span itemprop="startDate" content="{{ start_dt_iso }}"> {{ start_dt | format_date('long') }}</span>
     {% elif start_dt.year == end_dt.year and start_dt.month == end_dt.month %}
-        {{ start_dt.day }}-{{ end_dt.day }} {{ start_dt | format_date('MMMM yyyy') }}
+        <span itemprop="startDate" content="{{ start_dt_iso }}">{{ start_dt.day }}</span>
```

Comment:
there will be whitespace between this and the `-`. to avoid this, add this at the end of this line: ``{#--#}``

Reply:
Fixed